### PR TITLE
fix: in mobile view roadmap card  border fix

### LINKED ISSUE #6748: Roadmap formatting
### Description

The right border of the first card is missing, and the left border of the final card is missing on my mobile device. Eddie knows what I'm describing. I can add a screenshot if needed. It is on the new Roadmap page.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Lalman888**
## Fixes Issue

 Closes #6748 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6762"></a>

### CODE REVIEW

**Review by ChinmayMhatre — COMMENTED** [review]
Change looks good. I think formatting needs to be done with prettier

**Comment by ChinmayMhatre** [comment]
@Lalman888 Changes look good. As suggested please run prettier on these changes in vscode that should fix the formatting. Let me know if you need any help.

**Comment by Lalman888** [comment]
now done

File: pages/roadmap.js [hunk 1]
```diff
@@ -95,10 +95,11 @@ export default function Roadmap() {
                   key={phase.id}
                   className={classNames(
                     phase.mostPopular ? "lg:z-10 lg:rounded-b-none" : "lg:mt-8",
-                    phaseIdx === 0 ? "lg:rounded-r-none border-r-0 " : "",
+                    phaseIdx === 0 ? "lg:rounded-r-none lg:border-r-0" : "",
                     phaseIdx === releases.length - 1
-                      ? "lg:rounded-l-none border-l-0"
+                      ? "lg:rounded-l-none lg:border-l-0"
                       : "",
+
```
Comment (ChinmayMhatre) [inline pages/roadmap.js:102]:
```suggestion
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by ChinmayMhatre — APPROVED** [review]

**Comment by ChinmayMhatre** [comment]
Look Good!🙌🏻

**Comment by eddiejaoude** [comment]
Great collab 👍  thank you everyone

**Pull request merged**